server/service/src/asset/insert.rs

```diff
@@ -70,9 +73,32 @@ pub fn insert_asset(
                 }
                 None => input,
             };
+
             let new_asset = generate(input);
             AssetRowRepository::new(connection).upsert_one(&new_asset)?;
 
+            // Automatically create a location for this asset (if it's a cold chain asset)
+            if new_asset.asset_class_id == Some(COLD_CHAIN_EQUIPMENT_UUID.to_string()) {
+                if let Some(store_id) = new_asset.store_id {
```

Comment:
I'm wondering if user documentation is needed to state that locations are only automatically created when creating an asset at that store given that we can manage assets across all stores from OMS central server 

(Or can we just make locations for different stores automatically anyway without this check?)